CMakeLists.txt

```diff
@@ -331,7 +331,7 @@ if(OCOS_ENABLE_RE2_REGEX)
   endif()
 
   if(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
-    set_property(TARGET re2 PROPERTY COMPILE_OPTIONS)
+    set_property(TARGET re2::re2 PROPERTY COMPILE_OPTIONS)
```

Comment:
The error message is following, did you get it work in any lib in onnxruntime project?
CMake Error at CMakeLists.txt:334 (set_property):
  set_property can not be used on an ALIAS target.

Reply:
Yeah, it works there. This is strange.